src/utils/contracts.js

```diff
@@ -468,7 +467,7 @@ export async function setupContracts(account, library, chainId, fetchId) {
 }
 
 // calls to be executed only once after setup
-const setupCurve = async (curveAddressProvider, getContract, chainId) => {
```

Comment:
Does this cause a breakage in JS by changing arity?

Reply:
No, but it would be good to clean up callers